site/_generated/index.njk

```diff
@@ -35,7 +35,7 @@ hideNewsletter: true
   {% endif %}
 
   <!-- Chromebooks -->
-  <div id="home__chromebooks" class="wrapper wrapper--background wrapper--no-bottom wrapper--padded">{{ heroCta(l10n.home.chromebook.content, l10n.home.chromebook.image, l10n.home.chromebook.cta) }}</div>
+  <div id="home__banner" class="wrapper wrapper--padded wrapper--full-bleed wrapper__contained banner-interstitial__wrapper">{{ bannerInterstitial(l10n.home.chromebook.content, l10n.home.chromebook.image, l10n.home.chromebook.cta, l10n.home.chromebook.modifiers) }}</div>
```

Comment:
Something that stands out to me when using BEM is if one of the block's elements lives outside of a block. Why can't the `_wrapper` live inside of the contained component, maybe even as the top-level `block`?